app/templates/courses/invite-to-classroom-modal.jade

```diff
@@ -1,6 +1,8 @@
 extends /templates/core/modal-base-flat
 
 block modal-header-content
+  //- TODO: why doesn't this work when included in layout.static.pug?
+  script(src='https://www.google.com/recaptcha/api.js')
```

Comment:
I think we shouldn't try to do this anyway, in the general goal of only loading resources when we need them. A better setup would be to `npm install` whatever library we need and then `require` it, so that `api.js` is part of the teacher bundle. Or does recaptcha rely on the library being downloaded directly like this? If so, better to have the `google` service include a function which adds the script tag and resolves a Promise once the script tag loads, with `_.once` around it so that the script is only loaded the first time it's called.

I recommend either doing one of these things, or updating TODO to reflect these options.